make/mt-os/mt-stm32f4.mk

```diff
@@ -18,6 +18,7 @@ XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK_LWIP)/system
 XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK)/Middlewares/Third_Party/FreeRTOS/Source/include
 XI_COMPILER_FLAGS += -I$(XI_STM32_PATH_SDK)/Middlewares/Third_Party/FreeRTOS/Source/portable/GCC/ARM_CM4F/
 XI_COMPILER_FLAGS += -I$(LIBXIVELY)/examples/stm32/xively_demo_mt/Inc
```

Comment:
off: here libxively build relies on the example's Inc folder which is a unintended dependency. We have to remove it later. But not in this PR since it is different topic, I've just mentioned it for the record.